nginx/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM nginx:stable-alpine
+
+COPY nginx.conf /etc/nginx/templates/default.conf.template
```

Comment:
Copy Nginx configuration file to `/etc/nginx/templates` directory to embed environment variables.

> Using environment variables in nginx configuration (new in 1.19)
Out-of-the-box, nginx doesn't support environment variables inside most configuration blocks. But this image has a function, which will extract environment variables before nginx starts.
> ...
> By default, this function reads template files in `/etc/nginx/templates/*.template` and outputs the result of executing `envsubst` to `/etc/nginx/conf.d`
> ref https://hub.docker.com/_/nginx